### PR TITLE
fix: removing toc to fix the docs html

### PR DESCRIPTION
**Pull request by JessicaSachs**
This removes the TOC from the codegen. It's okay. We'll improve the Docs UX shortly.

Should solve #327

### CODE REVIEW

File: cypress/e2e/smoke.cy.ts [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
-  //   cy.visit('/playground/');
-  //   cy.contains('Playground under construction');
-  // });
+  it('renders this last code example in the code', () => {
+    // Click on any section in the sidebar
+    cy.get('@firstSectionLink')
+      .click()
+      // Make sure the number of code examples is the same between reloads
+      .get('.container pre code')
+      .then(($codeBlocks) => {
```
Comment (pkuczynski) [inline cypress/e2e/smoke.cy.ts:16]: Minor thing, but does cypress not support async/await?

Comment (JessicaSachs) [reply cypress/e2e/smoke.cy.ts:16]: No. It's a long story. These aren't promises, they're Cypress Chainables (Bluebird, essentially)

Comment (pkuczynski) [reply cypress/e2e/smoke.cy.ts:16]: Cypress has a lot of weird quirks to my taste ;) But anyway, thanks for explaining...

**Review by ST-DDT — APPROVED** [review]
Strange cause, but works.

File: cypress/e2e/smoke.cy.ts [hunk 2]
```diff
@@ -1,11 +1,25 @@
 describe('Smoke Test', () => {
-  it('compiles the guide page', () => {
+  beforeEach(() => {
     cy.visit('/guide/');
+    cy.get('a[href="/api/animal.html"]').as('firstSectionLink');
+  });
```
Comment (Shinigami92) [inline cypress/e2e/smoke.cy.ts:5]: **formatting:** just a missing empty line here

```suggestion
  });

```

File: cypress/e2e/smoke.cy.ts [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
+    // Click on any section in the sidebar
+    cy.get('@firstSectionLink')
+      .click()
+      // Make sure the number of code examples is the same between reloads
+      .get('.container pre code')
+      .then(($codeBlocks) => {
+        // Trigger a reload
+        cy.reload()
+          // Give the runtime a chance to update/fail
+          .wait(500)
```
Comment (Shinigami92) [inline cypress/e2e/smoke.cy.ts:20]: **suggestion (perf):** Do we have any chance to not use static time values here?
Maybe a wait on animation or something like this?

**Review by Shinigami92 — COMMENTED** [review]
I'm okay for now with this change/removal of toc, because in the sidebar you can still the what functions are there.

File: cypress/e2e/smoke.cy.ts (hunk 3 above)
Comment (Shinigami92) [reply cypress/e2e/smoke.cy.ts:21]: We will extract this into another issue, so we can merge this PR now

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**